src/main/charts/bitbucket/values.yaml

```diff
@@ -52,6 +52,17 @@ bitbucket:
     port: 80
     # -- The type of Kubernetes service to use for Bitbucket
     type: ClusterIP
+    annotations: {}
+  # -- Enable or disable an additional service for exposing SSH for external access.
+  # Disable when the SSH service is exposed through the ingress controller, or enable if the ingress controller does
+  # not support TCP.
```

Comment:
I think here we should link customers to the more substantial documentation describing Ingress configuraion that is going to be created.

Reply:
@nanux in this case it's EKS specific (which is fine). In our tests we create LoadBalancer svc and tell AWS a few additional details. Maybe we need an additional docs page (perhaps not part of this PR) to explain how clouds or standalone ingress controllers can/may handle ssh traffic, say, Nginx port forwarding etc. Maybe just give direction and briefly explain why a standard ingress object won't do?

Reply:
It is a good callout. We have discussed it today on our standup that we need to create a dedicated documentation and created https://bulldog.internal.atlassian.com/browse/DCKUBE-343

I will add your note there.